src/portal/components/NodesTable.vue

```diff
@@ -144,6 +144,7 @@ export default class NodesTable extends Vue {
       this.dNodeError = false;
       this.dNodeLoading = true;
       let res = await getFarmDetails(event.item.farm.id);
+      if (!res.length) throw new Error("Can't resolve farm data");
```

Comment:
as we discussed, it's not an error, it's enough to display a message e.g. `No farm` instead!

Reply:
it's impossible to have no farm there as we click on a node that comes from the farm.
this case was to check if there is some error that makes the farm_id = `undefined`, null, or some other errors like 502.
Anyway I will remove it from here as there is no way to verify if the res actually the desired array, and will check on status code on the `getFarmDetails`

Reply:
and if the status code = 200 and the array is empty there will be an error accessing `res[0].name` and will be handled by the catch

Reply:
> it's impossible to have no farm there as we click on a node that comes from the farm. this case was to check if there is some error that makes the farm_id = `undefined`, null, or some other errors like 502. Anyway I will check also on if the res is array

Always have farm_id which is the `uniquely identifies each record, so the case you mentioned will happen if you are going to read something that does not exist, and 502 is a status code while you chicking on it on the `getFarmsDetails` helper, `if it not == 200` then you'll get the result in the `catch` statement.

Reply:
so  i made the both checks on if the array is empty, and config axios to return an error if the status not 200